README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
@damithc 

Some of my thoughts, fwiw,

The name of the organization reflects its purpose. For me, the name "oss-generic" leads me to wonder (and worry) about the scope of the project.

Who/What is this project catering to? Is it catering to TEAMMATES, SE-EDU, PowerPointLabs? Or is it catering to the OSS community at large?

Semantic versioning, for example, caters to the OSS community at large. As such, the project has to deal with [116 open issues and 12 open PRs](https://github.com/mojombo/semver) at the time of writing. And that is only for a simple *version number*. Imagine if this project starts having workflow processes and coding standards.

Now, let's say this project caters to the OSS community at large. Let's assume it catches on, and other projects start adopting the coding standards/processes defined in the documents of this project. We are going to start having requests from other projects demanding us to tweak our documents to match their ideal workflow, or complain we change something which is at odds with their workflow. This means that *our* projects (se-edu, teammates etc.) will lose some degree of autonomy. 

Now, workflow processes contain a lot of human factors inside. For example, the Linux and Git projects still use emails since their contributors (at least, the ones that actually matter) prefer to use Email. This is at odds with what the general OSS communities want, which is to use GitHub. So, if our projects' workflow processes become beholden to the OSS community at large, we may start to have friction between what *our* contributors want and what the general OSS community wants. I would personally prefer our projects to have more flexibility and autonomy.

We could, of course, put our foot down and say "No, we are mainly serving our own projects", but then we will then get lots of unnecessary drama ;-)

Of course, all these problems won't happen if this project does not catch on with the wider OSS community, but then this would also not happen if the org was named something else.

So, I think that this project should make it very clear that it exists to *only serve* the TEAMMATES, SE-EDU, PowerPointLabs projects. This should be shown through its name (which should not imply that it is primarily catering to the OSS community at large). Of course, the wider OSS community is free to adopt them (as long as they credit us, a proper license will solve that), but I think that should not be the *main* purpose of this project.